dfanalyzer/main.py

```diff
@@ -268,15 +311,18 @@ def io_function(json_object, current_dict, time_approximate,condition_fn):
     if "args" in json_object:
         if "fhash" in json_object["args"]:
             if type(json_object["args"]["fhash"]) is str:
-                d["fhash"] = int(json_object["args"]["fhash"],16)
-            else: 
+                d["fhash"] = int(json_object["args"]["fhash"], 16)
+            else:
                 d["fhash"] = json_object["args"]["fhash"]
```

Comment:
`d["fhash"] = str(json_object["args"]["fhash"])`

We don't want to change it to int. Its a bug.